src/app/models/trip.js

```diff
@@ -0,0 +1,27 @@
+import Backbone from 'backbone';
+const Trip = Backbone.Model.extend({
+  validate(attributes) {
+    const errors = {};
+    if (!attributes.name) {
+      errors.name = ['cannot be blank'];
+    }
+    if (!attributes.category) {
+      errors.category = ['cannot be blank'];
+    }
+    if (!attributes.continent)  {
+      errors.continent = ['cannot be blank'];
+    }
+    if (!attributes.cost) {
+      errors.cost = ['cannot be blank'];
+    }
+    if (!attributes.weeks) {
```

Comment:
Consider also checking for non-numeric weeks and cost as well as continents that aren't in the list, although the radio button group will deal with that.